pytest_logger/plugin.py

```diff
@@ -195,6 +211,15 @@ def set_log_option_default(self, value):
         """ Sets default value of `log` option."""
         self._log_option_default = value
 
+    def divide_logs_by_result(self, div_by_res_logdir='div_by_res', div_by_res_subdirs=['failed']):
```

Comment:
Naming is unfortunate and inconsistent. Note that each user will need to write following names at least once somewhere:
```
method:              divide_logs_by_result
named param:         div_by_res_logdir
named param:         div_by_res_subdirs
default subdir name: div_by_res
```
Let's unpack this. 
1. Division of logs is not the goal, having them laid-out/filtered/sorted by outcome is closer to what user wants. I assume `by_outcome` is more accurate than `div`/`division`, especially since `outcome` is the term which pytest uses  for `failed`/`passed`/`skipped` set.
2. It makes little sense to use 3/4-word identifiers.
3. It's good to take scoping into account - named param can have almost any name (except keywords or builtins) without fear of colliding with anything, so `bla_bla_bla_foodir` is superfluous, use `foodir`.
4. It's not logdir, logdir is the thing called `logs` by default. It's the logdir's subdir. So let's call it subdir.
5. I think user would want to choose outcomes more frequently than the name of subdir, let's change the order of arguments.

I'd propose this:
```
def split_by_outcome(self, outcomes=['failed'], subdir='by_outcome')
```
which can be used this way:
```
logger_config.split_by_outcome()
logger_config.split_by_outcome(['failed', 'passed'])
logger_config.split_by_outcome(['failed', 'passed'], 'some_other_name')
logger_config.split_by_outcome(subdir='some_other_name')
```

Reply:
done as you proposed